board/chatbox.php

```diff
@@ -62,7 +62,7 @@ public function findTab()
 			$msgCountryID = 0;
 
 		// Enforce Global and Notes tabs when its not Regular press game.
-		if ( ($Game->pressType != 'Regular' && $Game->pressType != 'RulebookPress') && !(isset($Member) && $Member->countryID == $msgCountryID) )
+		if ( (($Game->pressType != 'Regular' && $Game->pressType != 'RulebookPress') || (isset($Game) && $Game->Members->isTempBanned())) && !(isset($Member) && $Member->countryID == $msgCountryID) )

```

Comment:
What will be the impact of this on someone in 2 games if they get forced into CD in 1 but not the other?

Reply:
isTempBanned returns true only if the user is temp banned and a member with status 'left', i.e. he is temp banned from *rejoining* this game. If you are still an active member of a game, there will be no effect. For a left game, the private chat will be deactivated as if it was viewed by user not participating.

Reply:
Okay gotcha.